recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
```

Comment:
I don't think there's a need to mention in the summary under what license this is provided. Two lines below that's also listed :wink: